README.md

```diff
@@ -480,6 +485,7 @@ Services to securely store your Docker images.
 * [Blog](http://jpetazzo.github.io/) of [@jpetazzo][jpetazzo]
 * [Blog](http://progrium.com/blog/) of [@progrium][progrium]
 * [Blog](http://sebgoa.blogspot.be/) of [@sebgoa][sebgoa]
+* [Blog](https://codefresh.io/) of [@codefresh](https://github.com/codefresh-io)
```

Comment:
According to our contribution guideline:

>If it's a tutorial or a blog post, they get outdated really quickly so we don't really put them on the list but if it is on a very advanced and/or specific topic, we will consider it!

I've randomly browsed the blog entries and, beside company/product announcements, I haven't found such advanced topic yet.